src/app/home/pages/PlansPage/components/Wizard/WizardContainer.tsx

```diff
@@ -141,12 +141,12 @@ const WizardContainer: React.FunctionComponent<IOtherProps> = (props: IOtherProp
         if (editPlanObj.status?.conditions) {
           editPlanObj.status.conditions?.forEach((element) => {
             if (element.type === 'MigrationTypeIdentified') {
-              switch (element.message) {
+              switch (element.reason) {
                 case 'StateMigrationPlan':
                   initialValuesCopy.migrationType = getMigrationType('state');
                   break;
                 case 'StorageConversionPlan':
-                  initialValuesCopy.migrationType = getMigrationType('state');
```

Comment:
dang, sorry I missed this before! good catch

Reply:
+1, thanks @pranavgaikwad